server/service/src/invoice_line/inbound_shipment_line/insert/generate.rs

```diff
@@ -68,9 +68,21 @@ fn generate_line(
         item_name,
         item_code,
         stock_line_id: None,
-        total_before_tax,
-        total_after_tax: total_after_tax(total_before_tax, tax),
-        tax,
+        total_before_tax: match total_before_tax {
```

Comment:
would prefer these to be done above in a variable, since you doing the same thing twice (it's easier to read if you just do it once at the top and less chance of error), i.e. when setting total_after_tax it would become:
total_after_tax: total_after_tax(total_before_tax)